pkg/core/types.go

```diff
@@ -51,3 +53,27 @@ type configObject struct {
 	allKeys  bool
 	keys     map[string]struct{}
 }
+
+type PodController interface {
```

Comment:
I wonder if this actually needs to be public, If we were to use the wrapper as I commented above, do you think this could be private?

Reply:
The PodController is now private in the new commit.